applications/IgaApplication/tests/cpp_tests/test_iga_shell_3p.cpp

```diff
@@ -50,6 +50,8 @@ namespace Testing
     // Tests the stiffness matrix of the Shell3pElement with a polynomial degree of p=3.
     KRATOS_TEST_CASE_IN_SUITE(IgaShell3pElementP3, KratosIgaFastSuite)
     {
+        KRATOS_SKIP_TEST_IF_NOT(KratosComponents<ConstitutiveLaw>::Has("LinearElasticPlaneStress2DLaw")) << "This test needs the StructuralMechanicsApplication" << std::endl;
```

Comment:
as i understand this means that these tests are currently being skipped. @rickyaristio @NickNick9 please take into account that they need to be moved to python ASAP

Reply:
this is being tracked in https://github.com/KratosMultiphysics/Kratos/issues/12326